packages/lesswrong/components/comments/CommentsListSection.tsx

```diff
@@ -22,7 +22,6 @@ const styles = theme => ({
     margin: "0px auto 15px auto",
     ...theme.typography.commentStyle,
 
-    background: "white",
```

Comment:
I'm not 100% sure what this was originally for. This looks like some kind of hack we applied to fix a ToC interaction or something (i.e. comments overlapping with ToC headings), and that if it's still necessary, there's probably some better way to achieve the goal.

I originally got rid of this while trying to change the PostsPage to be grey. I'm no longer doing that, but think it's best to get rid of the white background here and fix it later if it becomes a problem.